<launch>
    <node pkg="ros_pololu_servo" type="ros_pololu_servo_node" name="pololu_controller_node" output="screen">
        <param name="pololu_motors_yaml" value="$(find ros_pololu_servo)/launch/hare_ctrl_cfg.yaml" />
        <!-- <param name="port_name" value="/dev/pololu" />/dev/serial/by-id -->
        <param name="port_name" value="/dev/serial/by-id/usb-Pololu_Corporation_Pololu_Mini_Maestro_18-Channel_USB_Servo_Controller_00178552-if00" />

        <param name="rate_hz" value="10" />
        <param name="daisy_chain" value="false" />
    </node>

    <!-- HARE control node -->
    <node pkg="ros_pololu_servo" type="HARE_command.py" name="HARE_command" />

    <!-- joy node -->
    <node respawn="true" pkg="joy"
            type="joy_node" name="joy_node" >
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.12" />
    </node>
    
    <!-- joy to HARE conversion node -->  
    <param name="axis_linear" value="1" type="int"/>
    <param name="axis_angular" value="0" type="int"/>
    <param name="scale_linear" value="4000" type="double"/>
    <param name="scale_angular" value="4000" type="double"/>
    <node pkg="ros_pololu_servo" type="joystick_node" name="joystick2HARE"/>


</launch>
